### PR TITLE
bpo-44364:Add non integral tests for `sqrt()`

### PR DESCRIPTION
**Pull request by ajthr**
https://bugs.python.org/issue44364

### CODE REVIEW

File: Misc/NEWS.d/next/Tests/2021-06-09-15-32-05.bpo-44364.zu9Zee.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add non integral tests for `sqrt()` function.
```
Comment (mdickinson) [inline Misc/NEWS.d/next/Tests/2021-06-09-15-32-05.bpo-44364.zu9Zee.rst]:
```suggestion
Add non integral tests for :func:`math.sqrt` function.
```

**Review by mdickinson — APPROVED** [review]
LGTM - thank you! One minor suggested tweak to the news entry.

File: Misc/NEWS.d/next/Tests/2021-06-09-15-32-05.bpo-44364.zu9Zee.rst (hunk 1 above)
Comment (ajthr) [reply Misc/NEWS.d/next/Tests/2021-06-09-15-32-05.bpo-44364.zu9Zee.rst]: Done :)

**Comment by mdickinson** [comment]
Still LGTM. Will merge as soon as the checks complete. Thank you, and congratulations on your first contribution to CPython! (I'll follow up on your other PR shortly.)

**Comment by AjithRamachandran** [comment]
> Still LGTM. Will merge as soon as the checks complete. Thank you, and congratulations on your first contribution to CPython! (I'll follow up on your other PR shortly.)

ty :)

**Pull request merged**